crates/runtime/benches/bench.rs

```diff
@@ -185,6 +185,10 @@ async fn run_query_and_record_result(
     query_name: &str,
     query: &str,
 ) -> Result<(), String> {
+    // Additional round of query run before recording results.
```

Comment:
I would recommend to move this to Spark run method (if it is Spark related only)
```
pub(crate) async fn run(
    rt: &mut Runtime,
    benchmark_results: &mut BenchmarkResultsBuilder,
) -> Result<(), String> {
    let test_queries = get_test_queries();


    // wake up query

    for (query_name, query) in test_queries {
        super::run_query_and_record_result(rt, benchmark_results, "spark", query_name, query)
            .await?;
    }

    Ok(())
}
```

Reply:
It’s not just about spark, we want to discard first warm up run for all connectors